### ISSUE TITLE
Document how to start the Jupyter server

### ISSUE THREAD

**Issue by cuducos** [okfn-brasil/notebooks#3]

**Comment by jtemporal** [comment]
I think we should have instructions on
- running the jupyter server without docker
- running everything with docker (maybe even use a image for that)

**Comment by jtemporal** [comment]
@rodolfo-viana what do you think?

**Comment by rodolfo-viana** [comment]
@jtemporal, that is great! <3

**Comment by willianpaixao** [comment]
How about adding a section on "running with _conda_"?

**Comment by jtemporal** [comment]
@willianpaixao you mean how to do it when you only have anaconda installed?

**Comment by willianpaixao** [comment]
@jtemporal yes. Sounds silly, but you need one line to install the dependencies (basically the libs listed in requirements.txt) and one line to run Jupyter that comes with Anaconda.

**Comment by jtemporal** [comment]
It could be nice @willianpaixao I'd totally merge a PR with that instructions ^.^
